TheForceEngine/TFE_FrontEndUI/frontEndUi.cpp

```diff
@@ -2874,6 +2908,22 @@ namespace TFE_FrontEndUI
 		s32 framerate = (s32)system->gifRecordingFramerate;
 		DrawLabelledIntSlider(labelW, valueW - 2, "GIF Recording Framerate", "##CBO", &framerate, 10, 30);
 		system->gifRecordingFramerate = (f32)framerate;
+
+	#ifdef _WIN32
+		ImGui::Separator();
+		if (ImGui::Button("Open Log Folder"))
+		{
+			char logDir[TFE_MAX_PATH];
+			TFE_Paths::appendPath(TFE_PathType::PATH_USER_DOCUMENTS, "", logDir);
+			ShellExecute(NULL, "open", logDir, NULL, NULL, SW_SHOWNORMAL);
```

Comment:
You can just call `osShellExecute()` in system.h instead. That way you don't need to include Windows.h here.

Reply:
Updated.